pymc3/distributions/multivariate.py

```diff
@@ -1940,6 +1943,73 @@ def _distr_parameters_for_repr(self):
         return ["mu"]
 
 
+class CARRV(RandomVariable):
+    name = "car"
+    ndim_supp = 1
+    ndims_params = [1, 2, 0, 0]
+    dtype = "floatX"
+    _print_name = ("CAR", "\\operatorname{CAR}")
+
+    def make_node(self, rng, size, dtype, mu, W, alpha, tau):
+        mu = at.as_tensor_variable(mu)
```

Comment:
Should these be of type `mu = at.as_tensor_variable(floatX(mu))` and so on for the other variables?